P/Perl/build_tarballs.jl

```diff
@@ -141,7 +141,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Readline_jll"; compat="~8.1.1")
+    Dependency("Readline_jll"; compat="8.1.1")
```

Comment:
Actually, I am not sure this is enough: you'll also want to make sure you build against 8.1.1 -- otherwise there may be issues on macOS, where a .dylib built against readline 8.2.0 won't load it readline 8.1.1 is already loaded, even if it uses nothing that is exclusive to 8.2.0

Reply:
Excellent! I wasn't aware (anymore?!?) that `compat` is so clever, cool